src/amo/components/AddonMeta/index.js

```diff
@@ -48,14 +51,33 @@ export class AddonMetaBase extends React.Component<InternalProps> {
     let reviewTitle;
     if (!addon) {
       reviewCount = null;
-      reviewTitle = i18n.gettext('Ratings');
-    } else if (addonRatingCount) {
-      reviewCount = i18n.formatNumber(addonRatingCount);
-      reviewTitle = i18n.ngettext('Rating', 'Ratings', addonRatingCount);
+      reviewTitle = i18n.gettext('Reviews');
+    } else if (addonReviewCount) {
+      reviewCount = i18n.formatNumber(addonReviewCount);
+      reviewTitle = i18n.ngettext('Review', 'Reviews', addonReviewCount);
     } else {
-      reviewTitle = i18n.gettext('No Ratings');
+      reviewTitle = i18n.gettext('No Reviews');
     }
 
+    const reviewsLink =
+      addon && reviewCount ? `/addon/${addon.slug}/reviews/` : null;
+
+    const reviewsContent = reviewsLink ? (
+      <Link className="AddonMeta-reviews-content-link" to={reviewsLink}>
+        {reviewCount}
+      </Link>
+    ) : (
+      reviewCount
+    );
+
+    const reviewsTitle = reviewsLink ? (
+      <Link className="AddonMeta-reviews-title-link" to={reviewsLink}>
```

Comment:
Hmm. I thought I fixed the centering for this but I guess not.

Your patch:

<img width="188" alt="screenshot 2018-08-14 10 46 39" src="https://user-images.githubusercontent.com/55398/44102466-5bf2e844-9faf-11e8-99bc-780a1311f43b.png">


Could you add some centering? TBH I'm not sure why `justify-content: center` doesn't already do that. I guess it only applies to the block, not the text.

```diff
diff --git a/src/amo/components/AddonMeta/styles.scss b/src/amo/components/AddonMeta/styles.scss
index 0a8fb8ecb..58dd7df8a 100644
--- a/src/amo/components/AddonMeta/styles.scss
+++ b/src/amo/components/AddonMeta/styles.scss
@@ -41,6 +41,7 @@
 .AddonMeta-rating-title {
   display: flex;
   justify-content: center;
+  text-align: center;
 }
 
 .AddonMeta .MetadataCard-content,
```

<img width="178" alt="screenshot 2018-08-14 10 47 14" src="https://user-images.githubusercontent.com/55398/44102495-6c77ff10-9faf-11e8-9f3d-999ced1325ec.png">